shotover-proxy/tests/redis_int_tests/basic_driver_tests.rs

```diff
@@ -1216,19 +1219,21 @@ async fn test_source_tls_and_single_tls() {
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
 async fn test_cluster_ports_rewrite() {
-    let _compose = DockerCompose::new("examples/redis-cluster-ports-rewrite/docker-compose.yml")
-        .wait_for_n("Cluster state changed", 6);
+    let _compose =
+        DockerCompose::new("tests/test-topologies/redis-cluster-ports-rewrite/docker-compose.yml")
```

Comment:
Why cant we use the same config for testing?

If they really need a seperate test specific config we still need to test the example version in some way otherwise it will bitrot.

Reply:
We can but we would need to change our `DockerCompose` implementation to be able to mount files. Off the top of my head changing the directory so `$PWD` in the `docker-compose.yml` makes sense.

It could be useful in the future if we need to test more elaborate `docker-compose.yml` setups with Shotover in the future?

Might be out of scope for this PR though.